src/features/game/expansion/components/resources/Tree.tsx

```diff
@@ -187,64 +194,77 @@ export const Tree: React.FC<Props> = ({ treeIndex, expansionIndex }) => {
-            widthFrame={266 / 4}
-            heightFrame={168 / 4}
-            fps={24}
-            steps={7}
-            direction={`forward`}
-            autoplay={false}
-            loop={true}
-            onLoopComplete={(spritesheet) => {
-              spritesheet.pause();
-            }}
-          />
+        <>
           <div
-            className={`absolute bottom-8 -right-[1rem] transition pointer-events-none w-28 z-20 ${
-              showLabel ? "opacity-100" : "opacity-0"
-            }`}
+            onMouseEnter={handleHover}
+            onMouseLeave={handleMouseLeave}
+            ref={treeRef}
+            className="group cursor-pointer w-full h-full"
+            onClick={shake}
           >
-            <Label className="p-2">Equip {tool.toLowerCase()}</Label>
+            <Spritesheet
+              className="relative group-hover:img-highlight pointer-events-none transform w-full h-full"
+              style={{
+                width: `${SHAKE_SHEET_FRAME_WIDTH * PIXEL_SCALE}px`,
+                height: `${SHAKE_SHEET_FRAME_HEIGHT * PIXEL_SCALE}px`,
+                imageRendering: "pixelated",
+                position: "absolute",
+
+                // Adjust the base of tree to be perfectly aligned to
+                // on a grid point.
+                bottom: `${8 * PIXEL_SCALE}px`,
+                right: `-${4 * PIXEL_SCALE}px`,
+              }}
+              getInstance={(spritesheet) => {
+                shakeGif.current = spritesheet;
+              }}
+              image={shakeSheet}
+              widthFrame={SHAKE_SHEET_FRAME_WIDTH}
+              heightFrame={SHAKE_SHEET_FRAME_HEIGHT}
+              fps={24}
+              steps={7}
+              direction={`forward`}
+              autoplay={false}
+              loop={true}
+              onLoopComplete={(spritesheet) => {
+                spritesheet.pause();
+              }}
+            />
           </div>
-        </div>
+          <Overlay target={treeRef.current} show={showLabel} placement="right">
+            {(props) => (
+              <div {...props} className="absolute -left-1/2 z-10 w-28">
+                <Label className="p-2">Equip {tool.toLowerCase()}</Label>
+              </div>
+            )}
+          </Overlay>
+        </>
       )}
 
       <Spritesheet
         style={{
-          width: `${GRID_WIDTH_PX * 4}px`,
-          // Line it up with the click area
-          transform: `translateX(-${GRID_WIDTH_PX * 2.5}px)`,
           opacity: collecting ? 1 : 0,
           transition: "opacity 0.2s ease-in",
+
+          width: `${CHOPPED_SHEET_FRAME_WIDTH * PIXEL_SCALE}px`,
+          height: `${CHOPPED_SHEET_FRAME_HEIGHT * PIXEL_SCALE}px`,
           imageRendering: "pixelated",
+          position: "absolute",
+
+          // Adjust the base of tree to be perfectly aligned to
+          // on a grid point.
+          bottom: `${8 * PIXEL_SCALE}px`,
+          right: `-${4 * PIXEL_SCALE}px`,
```

Comment:
The base of the tree sits on the intersection of two grid lines. Bit hard to see in the screenshot.

![base](https://user-images.githubusercontent.com/41215134/184062119-23eb7106-d44e-4f69-b680-d1bc16d5fe53.png)